js/main.js

```diff
@@ -1,2 +1,91 @@
 $(function () {
+  let random;
+
+  let stringBotScore = $("#computerScore").text()
+  let computerScore = parseInt(stringBotScore);
+  computerScore = 0;
+
+  let stringPlayerScore = $("#humanScore").text()
+  let humanScore = parseInt(stringPlayerScore);
+  humanScore = 0;
+
+
+  $("#rock").click(() => {
+  let result = compare("rock", botChoice());
+  console.log(result)
+  console.log(botChoice())
+  console.log(updateBotCount())
+  console.log(updatePlayerCount())
+  $("#status").html(result)
+});
+
+  $("#paper").click(() => {
+  let result = compare("paper", botChoice());
+  console.log(result)
+  console.log(botChoice())
+  console.log(updateBotCount())
+  console.log(updatePlayerCount())
+  $("#status").html(result)
+});
+
+
+  $("#scissors").click(() => {
+  let result = compare("scissors", botChoice());
+  console.log(result)
+  console.log(botChoice())
+  console.log(updateBotCount())
+  console.log(updatePlayerCount())
+  $("#status").html(result)
+});
+
+
+      const compare = (human, bot) => {
```

Comment:
@cbrooks2019 same issue here, your indentation is off which makes it difficult to navigate and spot bugs